**Brief Introduction:**
- I accept Scientist | Researcher | Specialist | System Administrator. [Curriculum Vitae | Past Employers References](https://drive.google.com/file/d/1Fl3VkXVq9eFir_DzXnavfCCZK9GyZo9a/view?usp=sharing) 
- My abilities span across bioinformatics, machine and deep learning, and computing system management. Lead data analyst/bioinformatician in all the workplaces. 
- From 2024 onwards, code stack  reduced to BASH, Awk, Python, GO and RUST, spanning across bioinformatics, data analysis, deep learning to system programming. 
- Lead role in development of the computing cluster and established and managed computing clusters. 
- Extremely collaborative,communicative, approachable, motivated, solution oriented, responsive, and initiative taker.
- I dont use AI for code writing and write at [Linux community](https://linuxcommunity.io/u/gauravearn/activity). 

**Work Environment:**
- If you have listened from me then it has value to you and me and that expresses my interest and willingness and if you have listened from someone else anything fake and not confirmed with me, it has nothing to do with me. 
- I am not responsible for anyone suggesting any personal/work preferences/establishing any sort of categorical/numerical association with anything without informing me, saying that I like that. You can ask me anytime and i will let you know. 
- I dont entertain a single word of listening someone personal preferences/likes/dislikes and neither I accept a personal relation from previous co-authors and nor i am interested in hearing from anyone with whom or whom not i should work or go around.  

**Employment: Bioinformatics | Deep Learning | System Administration**
- 2024-: **Universitat Potsdam,Germany, not associated with any group at present and self-learnt GO, RUST, benchmarked PacBioHifi and coded Python, GO, RUST.** 
- 2021-2023: Machine and Deep learning, Devops and self-learnt Python, Web and Application development. 
- 2020-2021: Finnish Museum of Natural History, Finland. I completed genome assembly, annotations, markers genes, ITS, and phylogenomics of over 500 fungal species using NextSeq.
- 2017-2021: Finnish Museum of Natural History, Finland, where I completed pacbio sequencing, assembly, annotations, organelle genomes and evolutionary/phylogenomic/comparative genomics approaches of lower plants *Coleochaete orbicularis*, *Blasia pusilla*, *Chaetospiridium orbicularis*, *Polytrichum commune*, *Mallomonas*, and *Cryptomonas*. Additionally, I worked for collaboration with Edinburgh UK, to analyse the PAFTOL species and the chloroplast genomes of the *Ambrosia* clade from Norway.
- 2016-2017: University of Connecticut, USA, where I analyzed the Douglas fir genome from genome annotation to the phylogenomics.
- 2014â€“2016: University of Technology, Sydney, Australia, where I developed computational methods for seagrasses.
- 2011-2014: Fondazione Edmund Mach, Italy, where I developed and analyzed bioinformatics methods for nuclear and organelle genomes, metagenomics from fungal and bacterial both whole genome as well as 16S and ITS sequencing.

**Code Stack:** 
- All code and ideas concieved and written by me unless specified. 
- Past:C++(dropped 2021) | Present: Bash, Awk, Python, GO, RUST.
- Machine | Deep Learning: PyTorch, Tensorflow, XGBoost, Scikit-learn, Keras, Fastai, Markdown, MkDocs, Quatro.
- System Analyst: Git,Terraform, Kubernetes, Grafana, LDAP, Systemd, Apptainer, Docker, Podman. 
- HPC/Database: SQlite3, PostgresSQL, PBS, SLURM, Fedora.
- Web Develpoment: Shiny/ShinyExpress(Python), Streamlit, Django, HUGO, Flask, HTMx. 
