**Brief Introduction:**

- Open/Accept position on Scientist | Researcher | Specialist | System Administrator. 
- [Curriculum Vitae/Past Employers References](https://github.com/codecreatede/codecreatede/blob/main/Curriculum_Vitae_Gaurav_Sablok_2024.pdf) | [Scopus](https://www.scopus.com/authid/detail.uri?authorId=36633064300) | [ORCID](https://orcid.org/0000-0002-4157-9405) | [Web of Science](https://www.webofscience.com/wos/author/record/C-5940-2014). 
- Extremely collaborative, Highly Communicative, approachable, motivated, task and solution oriented, time responsive, and initiative taker.
- I dont use AI for code writing and write at [Linux community](https://linuxcommunity.io/). [This will help you understand real person](https://drive.google.com/file/d/1pOfBWigcJAuQitLSdpxnD1YZ3hB6ZkU0/view?usp=sharing). 
-  If you havent heard from / confirmed with me and believed anything false, it has nothing to do with me. If someone has said i like anyone/mylikes/mydislikes/association with anything, kindly ignore such people or go to them and dont approach me. They are wasting your time and that has nothing to do with me. I dont allow someone to make a decision for me. Dont waste my and others time. 

**Work Areas/Employment: Bioinformatics | Deep Learning | System Administration**
- **2024 - :** Universitat Potsdam,Germany, where I self-learnt Julia, GO and Javascript, benchmarked PacBioHifi genome analysis and coded packages in Python, Ruby, Julia, GO. 
- **2021-2023:** Machine and Deep learning, Devops and self-learnt Python, Web and Application development. 
- **2020-2021:** Finnish Museum of Natural History and the University of Helsinki, Finland. I managed the sequencing used the NextSeq methods, including sequencing library methods. I completed genome assembly, annotations, markers genes, ITS, and phylogenomics of over 500 different species.
- **2017-2021:** Finnish Museum of Natural History and University of Helsinki, Finland, where I conducted bioinformatics and sequencing the genomes of lower plants *Coleochaete orbicularis*, *Blasia pusilla*, *Chaetospiridium orbicularis*, *Polytrichum commune*, *Mallomonas*, and *Cryptomonas* species, for which I did pacbio sequencing, assembly, annotations, organelle genomes and evolutionary/phylogenomic/comparative genomics approaches. Additionally, I worked for collaboration with the host organization such as Edinburgh UK, to analyse the genomics data for PAFTOL species and the chloroplast genomes of the *Ambrosia* clade from Norway.
- **2016-2017:** University of Connecticut, USA, where I analyzed the Douglas fir genome from the genome annotation to the phylogenomics and identifying genes and evolution of importance.
- **2014â€“2016:** University of Technology, Sydney, Australia, where I developed computational methods for seagrasses.
- **2011-2014:** Fondazione Edmund Mach, Italy, where I developed and analyzed bioinformatics methods for nuclear and organelle genomes, metagenomics from fungal and bacterial both whole genome as well as 16S and ITS sequencing.

**Scientific Stack:** 
- All code and ideas concieved and written by me unless specified. 
- **Till 2024**: C++, Bash, Awk, R, Python, Ruby, Julia, Javascript, GO. 
- **Current**: Bash/Shell, Awk, Python, Julia, Javascript, GO 
- **Machine | Deep Learning**: PyTorch, Tensorflow, XGBoost, Scikit-learn, Keras, LightGBM, Fastai. 
- **Package | API Development**: Python, Julia, GO, FASTAPI.
- **Documentation**: Markdown, MkDocs, Quatro.
- **System Analyst**: Git,Terraform, Kubernetes, Grafana, LDAP, Systemd, Apptainer, Docker, Podman. 
- **Database**: SQlite3, PostgresSQL, MongoDB. 
- **HPC**: PBS, SLURM.
- **Web Develpoment**: Electron, Shiny/ShinyExpress(Python), Streamlit, Django, HUGO, Flask, Javascript
- **OS|Code Editor|Terminal**: RedHat(Fedora), NeoVim, Cursor, Wezterm, tMUX, Zellij.
- **In case of golang, please see the last commit message for the final release.**
- **System Adminstration** - **Python** | [Python Packages](https://pypi.org/user/gauravcodepro/) |  [Slurm generator](https://github.com/codecreatede/universitat-potsdam-devops) | **Ruby** | [Ruby gems](https://rubygems.org/profiles/gauravcodepro) |  [Devopsutils](https://github.com/codecreatede/devops-system-profiler) | [Ruby-template](https://github.com/codecreatede/ruby-gem-create) | **Go** [Apptainer-devops](https://github.com/codecreatede/golang-apptainer-devops) | [Fedora-devops](https://github.com/codecreatede/fedora-devops) | [GO-LDAP](https://github.com/codecreatede/golang-ldap) | [GO-Network](https://github.com/codecreatede/go-network-analyzer) | [Network-analyzer](https://github.com/codecreatede/go-network-analyzer) | [TCP-analyzer](https://github.com/codecreatede/tcp-analyzer)
- **Bioinformatics** - **Python** | [TAIR Streamlit](https://githb.com/arabidopsis-genome-analyzer) |[Tairaccession](https://github.com/codecreatede/tairaccession)| [Metabolic-analyzer](https://github.com/codecreatede/BIGG-metabolic-analyzer-API) | [Metabolic-json](https://github.com/codecreatede/metabolic-json) | [Ontology-graphs](https://github.com/codecreatede/ontology-graphs) | [Pacbiohifi](https://github.com/codecreatede/pacbiohifi-desktop) | **Ruby** [Panacheextract](https://rubygems.org/gems/panacheextract) | [Protalign](https://github.com/codecreate/proteinalignment-annotation-gem) and [Proteinmultialign](https://github.com/codecreatede/protein-multialign-gem) | [Pacbiohifi-analyzer](https://github.com/codecreatede/pacbiohifi-analyzer) **Go** [block-estimate](https://github.com/codecreatede/go-alignment-block-estimate) | [Block-getter](https://github.com/codecreatede/go-alignment-block-getter) | [alignment-proportion](https://github.com/codecreatede/go-alignment-proportion) | [Alignment-estimate](https://github.com/codecreatede/go-alignment-estimate) | [Phylogenomics-Tab](https://github.com/codecreatede/go-phyloegenomics-tab) | [GO-metagenomics](https://github.com/codecreatede/go-metagenome-placer) | [Phylogenomics-filter](https://github.com/codecreatede/go-phylogenomics-filter) | [Alignment-merger](https://github.com/codecreatede/go-alignment-merger) | [GO-graphics](https://github.com/codecreatede/go-graphics) | [GO-Pacbiohifi](https://github.com/codecreatede/go-pacbiohifi) | [GO-mapper](https://github.com/codecreatede/go-mapper-diamond) | [GO-Pacbiohifi-Visualization](https://github.com/codecreatede/go-pacbiohifi-visualization)
