**About me**

**Plant, Bacterial, Fungal Bioinformatics, Deep Learning,and DevOPs.** [**Curriculum Vitae/Past Employers References**](https://github.com/gauravcodepro/gauravcodepro/blob/main/Curriculum_Vitae_Gaurav_Sablok_2024.pdf), [**Scopus**](https://www.scopus.com/authid/detail.uri?authorId=36633064300), [**ORCID**](https://orcid.org/0000-0002-4157-9405), [**Web of Science**](https://www.webofscience.com/wos/author/record/C-5940-2014).

**Previous repository [gauravcodepro](https://github.com/codecreatede), which i am migrating here this week for better code development and objectives.**

**Brief Introduction:**

- I have a background in bioinformatics and plant biology and extensive experience in code development using several programming languages(prior to 2021 - C++, Bash, AWK, R - after 2021: Python, Ruby, Julia, Javascript, GO), data analysis and data science, machine and deep learning and web and application development. I am open to species analysis at the genome level and also to enhance the knowledge gain. I have expertise motivating individuals with various mindsets to strive toward their objectives and get the desired results. During my time conducting research, I developed the skills I would need as a future scientist, including designing biological hypotheses, analyzing deep sequencing data, and interpreting its biological significance in the context of planned experiments. I consider myself to be someone with a high level of perseverance who is driven and eager to learn everything. My core teaching objective will be to help the students learn to actively think and pursue their independent thinking.

- As I worked on these plant, bacterial and fungal species, I created new computational analyses that helped identify the genes involved in abiotic and biotic interactions as well as unique genetic pathways that could help these species genetic breeding. In addition to developing bioinformatics to clarify functional genomics.I work from the bioinformatics to machine and deep learning. Additionally, the inclusion of the web and the application development using the python and javascript with added skills of deployment using the devops makes me a versatile and an approachable candidate. This continuous increase in my abilities and my constant efforts to put results are a proof of itself that I am a person, who is result and goal oriented.

**Academic/Research:**
- 2024 onwards, I worked at Universitat Potsdam, Germany, where I am currently based and working on my code and approach development, submitting applications for position and self-learnt Julia, GO and Javascript, adding Java (if needed) and approaches for machine and deep learning. During the time, I benchmarked PacBioHifi genome analysis and also coded several approaches, packages, gems in Python, Ruby, Julia, GO.
- 2022-2023: i added several new skills as a carrier advancement and added machine and deep learning, added several certification on devops and self-learnt Python,Web and Application development. I am also self-learning ArcPy to link the geospatial analysis and deep learning approaches to merge the analysis.
- 2020-2021: Examining the genomes of fungi Finnish Museum of Natural History and the University of Helsinki, Finland ,species have been sequenced using NextSeq methods. This work is concentrated on genome assembly, annotations, markers genes, and phylogenomics of those fungi. I have assembled, annotated, and identified ITS and other phylogenomics markers, as well as performed alignments, phylogenies, and downstream analyses on the fungal genomes of over 500 different species.
- 2017-2021: Postdoctoral researcher at the Finnish Museum of Natural History and the University of Helsinki, Finland conducting research on genome bioinformatics and sequencing the genomes of lower plants, including Coleochaete orbicularis, Blasia pusilla, Chaetospiridium orbicularis, Polytrichum commune, Mallomonas, and Cryptomonas species. I focused on genome assembly, genome annotation, chloroplast genomics, and a variety of other topics. Additionally, I've worked for various other organisations, collaborating such as Edinburgh UK, to analyse the genomics data for PAFTOL species and the chloroplast genomes of the Ambrosia clade from Norway.
- 2016-2017: I spent a short time spent at University of Connecticut, USA, where i analyzed the Douglas fir genome from the genome annotation to the phylogenomics and identifying genes and evolution of importance.
- 2014â€“2016: Research Fellow (Academic Level B) at the University of Technology, Sydney, Australia, where I developed computational methods for understanding seagrasses.
- 2011-2014: I developed and analyzed bioinformatics methods for transcriptional and post-transcriptional genomics across nuclear and organelle genomes at Fondazione Edmund Mach, Italy. I analyzed and finished multiple RNA-seq and Organelle-Seq experiments for several plant species. I independently created international partnership to find and created computational methods for a number of crop species. Additionally i analyzed multiple metagenomics anlaysis coming from the fungal and bacterial species involving ITS metagenomics, as well as the bacterial metagenomics.

**Areas of Interest:**
- All code and ideas concieved and written by me unless specified. Proficient in solving limitations. I select and drop language as per the position, if position focussed on data analysis and deep learning, i drop DevOPs and if DevOPs then i keep only mainstream programming language.
- **Bioinformatics,Deep Learning including Language Models and Graphs.** : Bash, Python, R, Ruby, Julia, Javascript, Awk, GO **Machine/Deep Learning**: PyTorch, Tensorflow, XGBoost, Scikit-learn, Keras, LightGBM, Fastai. **Package Development**: Python, Julia, Ruby, GO **API Development**: FASTAPI. **Documentation**: Markdown, ReadTheDocs, MkDocs, Jekyll, Github Pages, Quatro.
- **DevOPs**: Bash, Awk, Python, GO. **Code Management**: Git, GitKraken, Docker **Orchestration**:Terraform, Kubernetes. **Visualization**: Grafana. **Database**: SQlite3, MySQL, MongoDB. **HPC**: PBS and SLURM
- **Web Develpoment**: Rails, Python Shiny/ShinyExpress, Streamlit, Django, Julia, Flask, Javascript, Typescript
- **Application Development**: Electron(Javascript), PyQT(Python), Fyne(GO)

**Package/Application Development:**
- [**Python Packages**](https://pypi.org/user/gauravcodepro/) | [**Python Community**](https://www.python.org/community/)  ðŸ‘‰ [**Tairaccession**](https://github.com/gauravcodepro/tairaccession) | [**Graphanalyzer**](https://github.com/gauravcodepro/graphanalyzer) | [**RNAprocessor**](https://github.com/gauravcodepro/rnaprocessor) | [**ProteinAnnotator**](https://github.com/gauravcodepro/protein-annotator) | [**Ontologyanalyzer**](https://github.com/gauravcodepro/ontologyanalyzer)
- [**Streamlit Applications**](https://streamlit.io/) | [**Streamlit Community**](https://discuss.streamlit.io/) ðŸ‘‰ [**Metabolic-analyzer**](https://github.com/gauravcodepro/BIGG-metabolic-analyzer-API) | [**Metabolic-json**](https://github.com/gauravcodepro/metabolic-json-modelling) | [**Ontology-graphs**](https://github.com/gauravcodepro/arabidopsis-ontology-graphs) | [**Pacbiohifi**](https://github.com/gauravcodepro/pacbiohifi) | [**Slurm generator**](https://github.com/gauravcodepro/universitat-potsdam-devops-application)
- [**Ruby gems**](https://rubygems.org/profiles/gauravcodepro) | [**Ruby app**](https://www.ruby-forum.com/) | [**Ruby Community**](https://www.ruby-forum.com/) ðŸ‘‰ [**Devopsutils**](https://github.com/gauravcodepro/devops-system) | [**Ruby-template**](https://github.com/gauravcodepro/ruby_gem_creator) | [**Fasta-tokenizer**](https://github.com/gauravcodepro/pacbiohifi-motif-scanner) | [**Panacheextract**](https://rubygems.org/gems/panacheextract) | [**Protalign**](https://github.com/gauravcodepro/proteinalignment-annotation-gem) and [**proteinmultialign**](https://github.com/gauravcodepro/protein-multialign-gem) | [**Pacbiohifi-analyzer**](https://github.com/gauravcodepro/pacbiohifi-analyzer)
- **Electron Application** ðŸ‘‰ [**TAIR genome analysis**](https://github.com/gauravcodepro/arabidopsis-tair-application)

**Work Attitude/Professionalism:**
- Extremely collaborative and I only supports those who support me.
- Even AI prompt says ask me then i will tell, asking is the first way to initiate a open discussion/building relation.
- Highly Communicative, approachable, motivated, task and solution oriented, time responsive, and initiative taker.
- I believe decisions and communication not made in time have no and importance.
- I give importance and priority to the people I am working and communicating with.
- I believe that who is showing results is not fake whereas who is only talking with no results is fake.
- I am interested in all work and code related queries and position offers and not interested in anything else on this platform.
