**Brief Introduction: About me**

- Open/Accept position on Scientist/Academics/Researcher/Specialist/System Administrator. 
- [Curriculum Vitae/Past Employers References](https://github.com/codecreatede/codecreatede/blob/main/Curriculum_Vitae_Gaurav_Sablok_2024.pdf), [Scopus](https://www.scopus.com/authid/detail.uri?authorId=36633064300), [ORCID](https://orcid.org/0000-0002-4157-9405), [Web of Science](https://www.webofscience.com/wos/author/record/C-5940-2014). 
- Extremely collaborative and I only supports those who support me and take right approaches. Highly Communicative, approachable, motivated, task and solution oriented, time responsive, and initiative taker.
- I dont use AI for code writing and write at [Linux community](https://linuxcommunity.io/). 
- [This will help you understand real person](https://drive.google.com/file/d/1pOfBWigcJAuQitLSdpxnD1YZ3hB6ZkU0/view?usp=sharing). 
-  If you havent heard from me or confirmed with me and believed anything, it has nothing to do with me.
- If someone has said i like anyone/mylikes/mydislikes, kindly ignore such people or go to them and dont approach me. You are wasting your time by listening to such people and that has nothing to do with me. 
- I dont allow someone to make a decision for me. Dont dont waste my and others time. 

**Research:Plant, Bacterial, Fungal Bioinformatics, Deep Learning, DevOPs/System Administration.**
- **2024- :** Universitat Potsdam,Germany, where I am currently based and self-learnt Julia, GO and Javascript, Geospatial Application Development. I benchmarked PacBioHifi genome analysis with complete web-based real time monitoring invoking javascript/HTML-CSS, and coded several packages, gems in Python, Ruby, Julia, GO. 
- **2021-2023:** I added several new skills as a carrier advancement and added machine and deep learning, added certification on devops and self-learnt Python, Web and Application development. 
- **2020-2021:** Finnish Museum of Natural History and the University of Helsinki, Finland. I managed the sequencing used the NextSeq methods, including sequencing library methods. I completed genome assembly, annotations, markers genes, ITS, and phylogenomics of over 500 different species.
- **2017-2021:** Finnish Museum of Natural History and University of Helsinki, Finland conducting research on genome bioinformatics and sequencing the genomes of lower plants, including *Coleochaete orbicularis*, *Blasia pusilla*, *Chaetospiridium orbicularis*, *Polytrichum commune*, *Mallomonas*, and *Cryptomonas* species, in which I did pacbio sequencing, assembly, annotations, organelle genomes and evolutionary/phylogenomic/comparative genomics approaches. Additionally, during that tenure, I worked for collaborations with the host organization such as Edinburgh UK, to analyse the genomics data for PAFTOL species and the chloroplast genomes of the Ambrosia clade from Norway.
- **2016-2017:** University of Connecticut, USA, where i analyzed the Douglas fir genome from the genome annotation to the phylogenomics and identifying genes and evolution of importance.
- **2014â€“2016:** University of Technology, Sydney, Australia, where I developed computational methods for understanding seagrasses.
- **2011-2014:** Fondazione Edmund Mach, Italy, where I developed and analyzed bioinformatics methods for transcriptional and post-transcriptional genomics across nuclear and organelle genomes. I analyzed multiple metagenomics anlaysis coming from the fungal and bacterial species involving ITS metagenomics, as well as the bacterial genome metagenomics.

**Scientific/Application/System Administration:** 
- All code and ideas concieved and written by me unless specified. 
- Till 2024: C++, Bash, Awk, R, Bash, Awk, Python, Ruby, Julia, Javascript, GO. 
- **Bioinformatics,Deep Learning** : Bash/Shell, Awk, Python, Julia, Javascript, GO 
- **Machine/Deep Learning**: PyTorch, Tensorflow, XGBoost, Scikit-learn, Keras, LightGBM, Fastai. 
- **Package Development**: Python, Julia, GO **API Development**: FASTAPI. **Documentation**: Markdown, Quatro.
- **System Administration**: Bash, Awk, Python, GO. **Code Management**: Git, GitKraken **Orchestration**:Terraform, Kubernetes. **Visualization**: Grafana. **LDAP**: LDAPManagement and Server configuration
- **Database**: SQlite3, PostgresSQL, MongoDB. **HPC and Containerization**: PBS, SLURM, Apptainer, Docker, Podman. 
- **Web Application Develpoment**: Shiny/ShinyExpress(Python), Streamlit, Django, Julia, HUGO, Flask, Javascript.
- **Geospatial Application Developement**: Python, Javascript
- **Application Development**: Javascript, Python, GO
- **Operating System and Code Editor**: Fedora, NeoVide, Cursor. 
- **Terminal:** Wezterm, tMUX, Zellij.

**Package/Application Development:**  In case of golang, please see the last commit message and if it is final release and binary attached then it final else build verion. 

- **SyStem Adminstration** - **Python** | [**Python Packages**](https://pypi.org/user/gauravcodepro/) |  [**Slurm generator**](https://github.com/codecreatede/universitat-potsdam-devops) | **Ruby** | [**Ruby gems**](https://rubygems.org/profiles/gauravcodepro) |  [**Devopsutils**](https://github.com/codecreatede/devops-system-profiler) | [**Ruby-template**](https://github.com/codecreatede/ruby-gem-create) | - **Golang** [**Apptainer-devops**](https://github.com/codecreatede/golang-apptainer-devops) | [**Fedora-devops**](https://github.com/codecreatede/fedora-devops) | [**go-ldap**](https://github.com/codecreatede/golang-ldap) | [golang-network](https://github.com/codecreatede/go-network-devops-systemd)

- **Bioinformatics** - **Python** | [**TAIR Streamlit**](https://githb.com/arabidopsis-genome-analyzer) [**Tairaccession**](https://github.com/codecreatede/tairaccession)| [**Metabolic-analyzer**](https://github.com/codecreatede/BIGG-metabolic-analyzer-API) | [**Metabolic-json**](https://github.com/codecreatede/metabolic-json) | [**Ontology-graphs**](https://github.com/codecreatede/ontology-graphs) | [**Pacbiohifi**](https://github.com/codecreatede/pacbiohifi-desktop) | **Ruby** [**Fasta-tokenizer**](https://github.com/codecreatede/pacbiohifi-motif-scanner) and [**Kmer-counter**](https://github.com/codecreatede/kmer-count) | [**Panacheextract**](https://rubygems.org/gems/panacheextract) | [**Protalign**](https://github.com/codecreate/proteinalignment-annotation-gem) and [**Proteinmultialign**](https://github.com/codecreatede/protein-multialign-gem) | [**Pacbiohifi-analyzer**](https://github.com/codecreatede/pacbiohifi-analyzer) **Golang** [**block-estimate**](https://github.com/codecreatede/go-alignment-block-estimate) | [**block-getter**](https://github.com/codecreatede/go-alignment-block-getter) | [**alignment-proportion**](https://github.com/codecreatede/go-alignment-proportion) | [**alignment-estimate**](https://github.com/codecreatede/go-alignment-estimate) | [**phylogenomics-tab**](https://github.com/codecreatede/go-phyloegenomics-tab) | [**go-metagenomics**](https://github.com/codecreatede/go-metagenome-placer) | [**phylogenomics-filter**](https://github.com/codecreatede/go-phylogenomics-filter) | [**alignment-merger**](https://github.com/codecreatede/go-alignment-merger) | [**go-graphics**](https://github.com/codecreatede/go-graphics) | [**go-pacbiohifi**](https://github.com/codecreatede/go-pacbiohifi) | [**go-mapper**](https://github.com/codecreatede/go-mapper-diamond) | [**go-pacbiohifi-visualization**](https://github.com/codecreatede/go-pacbiohifi-visualization)

