**Brief Introduction:**

- Extremely collaborative,communicative, approachable, motivated, solution oriented, responsive, and initiative taker.
- Open to Scientist | Researcher | Specialist | System Administrator. [Curriculum Vitae/Past Employers References](https://github.com/codecreatede/codecreatede/blob/main/Curriculum_Vitae_Gaurav_Sablok_2024.pdf) 
- I dont use AI for code writing and write at [Linux community](https://linuxcommunity.io/). [This will help you understand real person](https://drive.google.com/file/d/1pOfBWigcJAuQitLSdpxnD1YZ3hB6ZkU0/view?usp=sharing). 
- If you have listened from me then it has value to you and me and that expresses my interest and willingness and if you have listened from someone else anything fake and not confirmed with me, it has nothing to do with me. 
- I am not responsible for anyone suggesting any personal/work preferences saying that I like that. You can ask me anytime and i will let you know.
- I dont entertain a single word of listening someone personal preferences/likes/dislikes. I dont accept a personal relation with any co-authors and neither i am interested in hearing from anyone with whom or whomnot i should work or go around.  

**Work Areas/Employment: Bioinformatics | Deep Learning | System Administration**
- 2024-: Universitat Potsdam,Germany, self-learnt Julia,GO, benchmarked PacBioHifi and coded Python, Ruby, Julia, GO. 
- 2021-2023: Machine and Deep learning, Devops and self-learnt Python, Web and Application development. 
- 2020-2021: Finnish Museum of Natural History, Finland. I managed the sequencing implemented NextSeq. I completed genome assembly, annotations, markers genes, ITS, and phylogenomics of over 500 different species.
- 2017-2021: Finnish Museum of Natural History, Finland, where I conducted bioinformatics and sequencing the genomes of lower plants *Coleochaete orbicularis*, *Blasia pusilla*, *Chaetospiridium orbicularis*, *Polytrichum commune*, *Mallomonas*, and *Cryptomonas* species, for which I did pacbio sequencing, assembly, annotations, organelle genomes and evolutionary/phylogenomic/comparative genomics approaches. Additionally, I worked for collaboration with the host organization such as Edinburgh UK, to analyse the genomics data for PAFTOL species and the chloroplast genomes of the *Ambrosia* clade from Norway.
- 2016-2017: University of Connecticut, USA, where I analyzed the Douglas fir genome from genome annotation to the phylogenomics.
- 2014â€“2016: University of Technology, Sydney, Australia, where I developed computational methods for seagrasses.
- 2011-2014: Fondazione Edmund Mach, Italy, where I developed and analyzed bioinformatics methods for nuclear and organelle genomes, metagenomics from fungal and bacterial both whole genome as well as 16S and ITS sequencing.

**Language Stack:** 
- All code and ideas concieved and written by me unless specified. 
- C++(dropped 2021),Bash, R, Ruby, Awk, Python, Julia, GO, RUST.
- Machine | Deep Learning: PyTorch, Tensorflow, XGBoost, Scikit-learn, Keras, Fastai. 
- Package/API Development  | Documentation: Python, Julia, GO, FASTAPI, Markdown, MkDocs, Quatro.
- System Analyst: Git,Terraform, Kubernetes, Grafana, LDAP, Systemd, Apptainer, Docker, Podman. 
- HPC/Database: SQlite3, PostgresSQL, MongoDB, PBS, SLURM, RedHat(Fedora).
- Web Develpoment: Shiny/ShinyExpress(Python), Streamlit, Django, HUGO, Flask, HTMx. 
- In case of golang, please see the last commit message for the final release.
- Golang and RUST binaries available from SourceForge [Source Forge](https://sourceforge.net/u/codecreatede/profile/)
- System Adminstration - [Python Packages](https://pypi.org/user/gauravcodepro/) |  [Slurm generator](https://github.com/codecreatede/universitat-potsdam-devops) | [Ruby gems](https://rubygems.org/profiles/gauravcodepro) |  [Devopsutils](https://github.com/codecreatede/devops-system-profiler) | [Ruby-template](https://github.com/codecreatede/ruby-gem-create) | [Apptainer-devops](https://github.com/codecreatede/golang-apptainer-devops) | [Fedora-devops](https://github.com/codecreatede/fedora-devops) | [GO-LDAP](https://github.com/codecreatede/golang-ldap) | [GO-Network](https://github.com/codecreatede/go-network-analyzer) | [Network-analyzer](https://github.com/codecreatede/go-network-analyzer) | [TCP-analyzer](https://github.com/codecreatede/tcp-analyzer)
- Bioinformatics - [TAIR Streamlit](https://githb.com/arabidopsis-genome-analyzer) |[Tairaccession](https://github.com/codecreatede/tairaccession)| [Metabolic-analyzer](https://github.com/codecreatede/BIGG-metabolic-analyzer-API) | [Metabolic-json](https://github.com/codecreatede/metabolic-json) | [Ontology-graphs](https://github.com/codecreatede/ontology-graphs) | [Pacbiohifi](https://github.com/codecreatede/pacbiohifi-desktop) | [Panacheextract](https://rubygems.org/gems/panacheextract) | [Protalign](https://github.com/codecreate/proteinalignment-annotation-gem) | [Proteinmultialign](https://github.com/codecreatede/protein-multialign-gem) | [Pacbiohifi-analyzer](https://github.com/codecreatede/pacbiohifi-analyzer) [block-estimate](https://github.com/codecreatede/go-alignment-block-estimate) | [Block-getter](https://github.com/codecreatede/go-alignment-block-getter) | [alignment-proportion](https://github.com/codecreatede/go-alignment-proportion) | [Alignment-estimate](https://github.com/codecreatede/go-alignment-estimate) | [Phylogenomics-Tab](https://github.com/codecreatede/go-phyloegenomics-tab) | [GO-metagenomics](https://github.com/codecreatede/go-metagenome-placer) | [Phylogenomics-filter](https://github.com/codecreatede/go-phylogenomics-filter) | [Alignment-merger](https://github.com/codecreatede/go-alignment-merger) | [GO-graphics](https://github.com/codecreatede/go-graphics) | [GO-Pacbiohifi](https://github.com/codecreatede/go-pacbiohifi) | [GO-mapper](https://github.com/codecreatede/go-mapper-diamond) | [GO-Pacbiohifi-Visualization](https://github.com/codecreatede/go-pacbiohifi-visualization)
