**Brief Introduction:**
- I would like to introduce myself as a person, who is very devoted towards writing good quality code and developing programs and analysis for bioinformatics, devops, system management, and cluster computing. I have lead all the bio-informatics analysis in all the projects till now without any support, and have written considerable code using BASH, R, Awk, Ruby, Python, Julia, GO. Since 2024, i have been actively coding in BASH, Awk, Python, Julia, GO, RUST. 
- I have played a active role in development of the computing facilities in my previous position and have established computing cluster and managed computing clusters. On the server side, I mainly used BASH and GO for the management of the cluster. I am well versed with the github actions and also integration of several devops tools, implementing new code approaches and also solving requirements limitations. I actively lead a continuous phase of learning and actively writing code spanning across from devops to bioinformatics analysis and machine and deep learning.  
- My wide skill set and extensive experience make me a strong candidate, who can play a variety of role across the selected position. 
- Extremely collaborative,communicative, approachable, motivated, solution oriented, responsive, and initiative taker.
- Open to Scientist | Researcher | Specialist | System Administrator. [Curriculum Vitae/Past Employers References](https://github.com/codecreatede/codecreatede/blob/main/Curriculum_Vitae_Gaurav_Sablok_2024.pdf) 
- I dont use AI for code writing and write at [Linux community](https://linuxcommunity.io/). 
- If you have listened from me then it has value to you and me and that expresses my interest and willingness and if you have listened from someone else anything fake and not confirmed with me, it has nothing to do with me. 
- I am not responsible for anyone suggesting any personal/work preferences saying that I like that. You can ask me anytime and i will let you know.
- I dont entertain a single word of listening someone personal preferences/likes/dislikes and neither I accept a personal relation from previous co-authors and nor i am interested in hearing from anyone with whom or whomnot i should work or go around.  

**Work Areas/Employment: Bioinformatics | Deep Learning | System Administration**
- 2024-: Universitat Potsdam,Germany, self-learnt Julia,GO, RUST benchmarked PacBioHifi and coded Python, Ruby, Julia, GO. 
- 2021-2023: Machine and Deep learning, Devops and self-learnt Python, Web and Application development. 
- 2020-2021: Finnish Museum of Natural History, Finland. I completed genome assembly, annotations, markers genes, ITS, and phylogenomics of over 500 fungal species using NextSeq.
- 2017-2021: Finnish Museum of Natural History, Finland, where I conducted pacbio sequencing, assembly, annotations, organelle genomes and evolutionary/phylogenomic/comparative genomics approaches of lower plants *Coleochaete orbicularis*, *Blasia pusilla*, *Chaetospiridium orbicularis*, *Polytrichum commune*, *Mallomonas*, and *Cryptomonas*. Additionally, I worked for collaboration with Edinburgh UK, to analyse the PAFTOL species and the chloroplast genomes of the *Ambrosia* clade from Norway.
- 2016-2017: University of Connecticut, USA, where I analyzed the Douglas fir genome from genome annotation to the phylogenomics.
- 2014â€“2016: University of Technology, Sydney, Australia, where I developed computational methods for seagrasses.
- 2011-2014: Fondazione Edmund Mach, Italy, where I developed and analyzed bioinformatics methods for nuclear and organelle genomes, metagenomics from fungal and bacterial both whole genome as well as 16S and ITS sequencing.

**Language Stack:** 
- All code and ideas concieved and written by me unless specified. 
- Past:C++(dropped 2021) | Now: Bash, Awk, Python, Julia, GO, RUST.
- Machine | Deep Learning: PyTorch, Tensorflow, XGBoost, Scikit-learn, Keras, Fastai, Markdown, MkDocs, Quatro.
- System Analyst: Git,Terraform, Kubernetes, Grafana, LDAP, Systemd, Apptainer, Docker, Podman. 
- HPC/Database: SQlite3, PostgresSQL, PBS, SLURM, RedHat(Fedora).
- Web Develpoment: Shiny/ShinyExpress(Python), Streamlit, Django, HUGO, Flask, HTMx. 
- Golang, please see the last commit message for the final release.
- [Golang and RUST binaries Source Forge](https://sourceforge.net/u/codecreatede/profile/)
- [Glimpse of topic sorted repositories](https://github.com/codecreatede/codecreatede/blob/main/topicsorted.md)
