**About me**

**I'm based in Germany/Deutschland.** You can contact me at **[gauravcodepro@gmail.com](mailto:gauravcodepro@gmail.com)** ü§ù¬†I'm open to Scientist/Academics/Researcher. [**Curriculum Vitae/Past Employers References**](https://github.com/codecreatede/codecreatede/blob/main/Curriculum_Vitae_Gaurav_Sablok_2024.pdf), [**Scopus**](https://www.scopus.com/authid/detail.uri?authorId=36633064300), [**ORCID**](https://orcid.org/0000-0002-4157-9405), [**Web of Science**](https://www.webofscience.com/wos/author/record/C-5940-2014). This is my new repository to which i have finished the transfer from the old one. 

- I have a background in plant biology/bioinformatics, code **(prior to 2021: C++, Bash, AWK, R - after 2021: Bash, AWK, Python, Ruby, Julia, Javascript, GO)**. **I am writing code more in the new one as compared to the old one**.**At the level of DevOPS, i only work on what is listed in package/application development.** **I am not interested in cybersecurity, or areas not listed**. I have developed the skills, I would need as a scientist/academician/researcher, including designing biological hypotheses, analyzing deep sequencing data, and interpreting its biological significance in the context of planned experiments. I consider myself to be someone with a high level of perseverance who is driven and eager to learn and apply what i have learnt. I am result oriented, directive and featured person, who endures developing the capabilities and continous development. As of now, i dont use AI for code writing as my approach is specific to specific roles.

**Getting to Know me ahead**
- You can read this to understand how to avoid fake and how to invest in real people [**getting to know ahead**](https://drive.google.com/file/d/15Ws7Eez0DT4Usxz7hwSdS5fSPsoIISiz/view?usp=sharing)

**Academic/Research:Plant, Bacterial, Fungal Bioinformatics, Deep Learning,and DevOPs.**
- **2024 onwards:** Universitat Potsdam, Germany, where I am currently based and working on my code development, submitting applications and self-learnt Julia, GO and Javascript and made complete web for previous project invoking javascript/HTML-CSS forms. During this time, I benchmarked PacBioHifi genome analysis with complete web-based real time monitoring, and coded several approaches, packages, gems in Python, Ruby, Julia, GO.
- **2022-2023:** I added several new skills as a carrier advancement and added machine and deep learning, added certification on devops and self-learnt Python, Web and Application development. I am also self-learning ArcPy to link the geospatial analysis and deep learning approaches to merge the analysis.
- **2020-2021:** Finnish Museum of Natural History and the University of Helsinki, Finland. I managed the species sequencing using the NextSeq methods, including which library to apply for the sequencing and methods. I completed genome assembly, annotations, markers genes, ITS, and phylogenomics of over 500 different species.
- **2017-2021:** Finnish Museum of Natural History and the University of Helsinki, Finland conducting research on genome bioinformatics and sequencing the genomes of lower plants, including *Coleochaete orbicularis*, *Blasia pusilla*, *Chaetospiridium orbicularis*, *Polytrichum commune*, *Mallomonas*, and *Cryptomonas* species, which involves pacbio sequencing, assembly, annotations, organelle genomes and evolutionary/phylogenomic/comparative genomics approaches. I worked on the genome assembly, genome annotation, chloroplast genomics. Additionally, during that tenure, I worked for various other organisations, collaborating with the host organization such as Edinburgh UK, to analyse the genomics data for PAFTOL species and the chloroplast genomes of the Ambrosia clade from Norway.
- **2016-2017:** University of Connecticut, USA, where i analyzed the Douglas fir genome from the genome annotation to the phylogenomics and identifying genes and evolution of importance.
- **2014‚Äì2016:** University of Technology, Sydney, Australia, where I developed computational methods for understanding seagrasses.
- **2011-2014:** Fondazione Edmund Mach, Italy, where I developed and analyzed bioinformatics methods for transcriptional and post-transcriptional genomics across nuclear and organelle genomes. I analyzed multiple metagenomics anlaysis coming from the fungal and bacterial species involving ITS metagenomics, as well as the bacterial genome metagenomics.

**Areas of Interest:**
- All code and ideas concieved and written by me unless specified. Proficient in solving limitations. 
- **Bioinformatics,Deep Learning including Language Models and Graphs.** : Bash, Python, R, Ruby, Julia, Javascript, Awk, GO **Machine/Deep Learning**: PyTorch, Tensorflow, XGBoost, Scikit-learn, Keras, LightGBM, Fastai. **Package Development**: Python, Julia, Ruby, GO **API Development**: FASTAPI. **Documentation**: Markdown, ReadTheDocs, MkDocs, Jekyll, Github Pages, Quatro.
- **DevOPs**: Bash, Awk, Python, GO. **Code Management**: Git, GitKraken, Docker **Orchestration**:Terraform, Kubernetes. **Visualization**: Grafana. **Database**: SQlite3, MySQL, MongoDB. **HPC**: PBS and SLURM
- **Web Develpoment**: Rails, Python Shiny/ShinyExpress, Streamlit, Django, Julia, Flask, Javascript, Typescript.
- **Application Development**: Electron(Javascript), PyQT(Python), Fyne(GO)
- **Operating System**: Ubuntu, Manjaro, Neovim, Neovide
- **Terminal:** GPU based: Wezterm,tMUX, Zellij for webassembly **LocalDev**: Tabby, Warp. 

**Package/Application Development:**
- [**Python Packages**](https://pypi.org/user/gauravcodepro/) | [**Python Community**](https://www.python.org/community/) | [**Streamlit Applications**](https://streamlit.io/) | [**Streamlit Community**](https://discuss.streamlit.io/) | [**Tairaccession**](https://github.com/codecreatede/tairaccession)| [**Metabolic-analyzer**](https://github.com/codecreatede/BIGG-metabolic-analyzer-API) | [**Metabolic-json**](https://github.com/codecreatede/metabolic-json) | [**Ontology-graphs**](https://github.com/codecreatede/ontology-graphs) | [**Pacbiohifi**](https://github.com/codecreatede/pacbiohifi-desktop) | [**Slurm generator**](https://github.com/codecreatede/universitat-potsdam-devops)

- [**Ruby gems**](https://rubygems.org/profiles/gauravcodepro) | [**Ruby app**](https://www.ruby-forum.com/) | [**Ruby Community**](https://www.ruby-forum.com/) [**Devopsutils**](https://github.com/codecreatede/devops-system-profiler) | [**Ruby-template**](https://github.com/codecreatede/ruby-gem-create) | [**Fasta-tokenizer**](https://github.com/codecreatede/pacbiohifi-motif-scanner) and [**Kmer-counter**](https://github.com/codecreatede/kmer-count) | [**Panacheextract**](https://rubygems.org/gems/panacheextract) | [**Protalign**](https://github.com/codecreate/proteinalignment-annotation-gem) and [**proteinmultialign**](https://github.com/codecreatede/protein-multialign-gem) | [**Pacbiohifi-analyzer**](https://github.com/codecreatede/pacbiohifi-analyzer)

**Work Attitude/Professionalism:**
- Extremely collaborative and I only supports those who support me.
- Even AI prompt says ask me then i will tell, asking is the first way to initiate a open discussion/building relation.
- Highly Communicative, approachable, motivated, task and solution oriented, time responsive, and initiative taker.
- I believe decisions and communication not made have no and importance.
- I give importance and priority to the people I am working and communicating with.
- I believe that who is showing results is not fake whereas who is only talking with no results is fake.
- I am very open to submitted applications and new positions and take active part in communication.
- Deciding my personal preferences/choices/likes/dislikes is not anyone role and neither i consider nor i give importance. 
- If you have heard anything, make an approach to confirm with me, may be you have heard from wrong or false people. 
