**Brief Introduction**
- Academics | Scientist | System Administrator | Bioinformatician. 
- Lead data bioinformatician in	all the employment. 
- Lead the development of the computing cluster and established and managed computing clusters. 
- Extremely collaborative,communicative, approachable, motivated, solution oriented, responsive, and initiative taker.
- I read and apply extensive data structures/algorithms. 

**Employment: Bioinformatics | Deep Learning | System Administration**
- [Curriculum Vitae | Past Employers References](https://drive.google.com/file/d/1Fl3VkXVq9eFir_DzXnavfCCZK9GyZo9a/view?usp=sharing) | [Scopus](https://www.scopus.com/authid/detail.uri?authorId=36633064300) | [Web of Science](https://www.webofscience.com/wos/author/record/1149035) | [Frontiers](https://loop.frontiersin.org/people/33293/overview)
- 2024-: Universitat Potsdam,Germany, not associated with any group at present and self-learnt GO, RUST, benchmarked PacBioHifi and coded Python, GO, RUST. 
- 2021-2023: Machine and Deep learning, DevOPS and self-learnt Python, Web and Application development. 
- 2020-2021: Finnish Museum of Natural History, Finland. I completed genome assembly, annotations, markers genes, ITS, and phylogenomics of over 500 fungal species using NextSeq.
- 2017-2021: Finnish Museum of Natural History, Finland, where I completed pacbio sequencing, assembly, annotations, organelle genomes and evolutionary/phylogenomic/comparative genomics approaches of lower plants *Coleochaete orbicularis*, *Blasia pusilla*, *Chaetospiridium orbicularis*, *Polytrichum commune*, *Mallomonas*, and *Cryptomonas*. Additionally, I worked for collaboration with Edinburgh UK, to analyse the PAFTOL species and the chloroplast genomes of the *Ambrosia* clade from Norway.
- 2016-2017: University of Connecticut, USA, where I analyzed the Douglas fir genome from genome annotation to the phylogenomics.
- 2014â€“2016: University of Technology, Sydney, Australia, where I developed computational methods for seagrasses.
- 2011-2014: Fondazione Edmund Mach, Italy, where I developed and analyzed bioinformatics methods for nuclear and organelle genomes, metagenomics from fungal and bacterial both whole genome as well as 16S and ITS sequencing.

**Bioinformatics | Deep Learning | System Adminstration Code Work Stack** 
- All code and ideas concieved and written by me unless specified and heavily inspired by data structures and algorithmic implementation, no AI based.
- Previous: C++, Current: Bash, Awk, Python, GO, RUST.
- Deep Learning: PyTorch, Tensorflow, Keras, XGBoost, Scikit-learn, Fastai, Markdown.
- System Analyst | HPC | Database: Git, Grafana, LDAP, Systemd, Apptainer, Docker, SQlite3, PostgresSQL, SLURM.
- Web Develpoment: Shiny/ShinyExpress, Streamlit, Django, Flask, HUGO, HTMx, Axum, Actix, Leptos. 

**Personal Approach**
- If you have listened from me then it has value to you and me and that expresses my interest and willingness and if you have listened from someone else anything fake and not confirmed with me, it has nothing to do with me. 
- I am not responsible for anyone suggesting any personal/work preferences/establishing any sort of categorical/numerical association with anything without informing me, saying that I like that. You can ask me anytime and i will let you know. 
- I dont entertain a single word of listening someone personal preferences/likes/dislikes and neither I accept a personal relation from previous co-authors and nor i am interested in hearing from anyone with whom or whom not i should work or go around.  
- Listen, believe and talk what is sensible and makes sense and confirm before believing as what you speak reflect yourself. 
